hawc/apps/common/table.py

```diff
@@ -0,0 +1,60 @@
+from html.parser import HTMLParser
+from typing import List
+
+from docx import Document
+from pydantic import BaseModel, conint, validator
+
+
+class GenericTableCell(BaseModel):
+    row: conint(ge=0)
+    column: conint(ge=0)
+    html_text: str
+
+    def to_docx(self, par):
+        parser = CellHTMLParser()
+        return parser.feed(self.html_text, par)
+
+
+class GenericTable(BaseModel):
+    rows: conint(gt=0)
+    columns: conint(gt=0)
+    cells: List[GenericTableCell]
+
+    @validator("cells")
+    def validate_cells(cls, cells, values):
```

Comment:
Add another to make sure one and only one cell for each position (may need to be in a global validator)

```python
d = set([(d.row, d.column) for d in cells])
if len(d) != tbl.rows * tbl.columns:
   raise ValueError(...)
```